CHANGES

```diff
@@ -20,7 +20,7 @@ Features added
 Bugs fixed
 ----------
 
-* #10678: Emit "source-read" events for RST files read in via `include` directive. 
+* #10678: Emit "source-read" events for RST files read in via ``include`` directive. 
```

Comment:
You could use <code>:dudir:\`include\`</code>.

Reply:
Thanks for the tip! Done.